validation/standard/standard-releases.toml

```diff
@@ -1,2 +1,2 @@
-standard_release_l1 = "op-contracts/v1.6.0"
+standard_release_l1 = "op-contracts/v1.8.0"
 standard_release_l2 = "op-contracts/v1.7.0-beta.1+l2-contracts"
```

Comment:
@mslipper do you know where these are referenced?

Reply:
they're not referenced anywhere afaik